app/components/categories.js

```diff
@@ -1,12 +1,16 @@
 import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
 import { A } from '@ember/array';
 import { action } from '@ember/object';
 import { inject as service } from '@ember/service';
 
 
 export default class CategoriesComponent extends Component {
-  store = service();
-  router = service();
+  @service store;
+  @service router;
+
+  @tracked categories = [];
+
   menuItems = A([
```

Comment:
I just realized that here is now where menuItems are... oops

in the template if `menuItems` was `this.menuItems`, that'd clue my unobservant self to be like "oh, that data is supposed to exist on the class" oops :(